### ISSUE TITLE
SSL tests fail on some systems (newer Debian, CentOS 8)

### ISSUE THREAD

**Issue by eserte** [dex4er/Starlight#4]
On some of my smoker systems the test suite fails (if IO::Socket::SSL is installed):
```
failed to listen to port 39237: No such file or directory
cannot open port: 127.0.0.1:39237 at /home/cpansand/.cpan/build/2020030718/Test-TCP-2.22-0/blib/lib/Test/TCP.pm line 53.
t/60ssl.t .................. 
Dubious, test returned 111 (wstat 28416, 0x6f00)
No subtests run 
failed to listen to port 35273: No such file or directory
cannot open port: 127.0.0.1:35273 at /home/cpansand/.cpan/build/2020030718/Test-TCP-2.22-0/blib/lib/Test/TCP.pm line 53.
t/61ssl_largebody.t ........ 
Dubious, test returned 111 (wstat 28416, 0x6f00)
No subtests run 
```
This happens on newer Debian systems (buster, bullseye) and CentOS 8.

**Comment by dex4er** [comment]
These systems reject SSL key (SHA1, 1K). I'll fix it in next release.

**Comment by dex4er** [comment]
fixed in b5533d17cc2b2d74970df8715d1fb5e414a4679e

**Comment by dex4er** [comment]
released as https://github.com/dex4er/Starlight/releases/tag/0.0400
